heroku.yml

```diff
@@ -0,0 +1,3 @@
+build:
+  docker:
+    web: Dockerfile
```

Comment:
I thought we dropped the docker stuff for the Heroku button option

Reply:
I'll remove this 😉